modules/websocket-client/src/index.ts

```diff
@@ -0,0 +1,209 @@
+import WebSocket from "./websocket";
+import { randomStringWithTimeStamp } from "@phenyl/utils";
+import {
+  GeneralRequestData,
+  GeneralResponseData,
+  SubscriptionRequest,
+  SubscriptionResult,
+  VersionDiff,
+  VersionDiffListener,
+  VersionDiffSubscriber,
+  WebSocketServerSubscriptionResultMessage,
+  WebSocketServerVersionDiffMessage,
+  WebSocketServerResponseDataMessage
+} from "@phenyl/interfaces";
+
+type Id = string;
+type WebSocketMessageEvent = { data: any; type: string; target: WebSocket };
+
+/**
+ * Universal WebSocket Client for PhenylWebSocketServer.
+ */
+export default class PhenylWebSocketClient implements VersionDiffSubscriber {
```

Comment:
I remove RestApiHandler from intefaces because I got this error.

```
Class 'PhenylWebSocketClient' incorrectly implements interface 'RestApiHandler<GeneralTypeMap>'.
  Property 'handleRequestData' is missing in type 'PhenylWebSocketClient' but required in type 'RestApiHandler<GeneralTypeMap>'.ts(2420)
```

Tell me if there is a better way.